### ISSUE TITLE
CLI behavior incorrectly invoked by importing ecstatic in ES modules

### ISSUE THREAD

**Issue by xiata** [jfhbrook/node-ecstatic#226]
Importing a module should not trigger the CLI.
This is due to module.parent being null from imports into ES modules.

node: 10.8.0

`node --experimental-modules example.mjs`

example.mjs:

```javascript
import ecstatic from 'ecstatic';
console.log(ecstatic);
```

Console:
```
ecstatic serving ... at http://0.0.0.0:8000
(and the typical log from console logging a constructor)
```

**Comment by jfhbrook** [comment]
ugh.

I have no interest in explicitly supporting experimental features (I think es modules are kinda dumb if I can be real) but feel free to submit a pull request.

**Comment by jfhbrook** [comment]
most likely the move is to just factor that code out into a cli.js and update npm accordingly. Relatively straightforward, I'm just lazy.

**Comment by xiata** [comment]
Was about to write up that suggestion, I'll try to get around to it this weekend

**Comment by jfhbrook** [comment]
Sounds good!

**Comment by jfhbrook** [comment]
Should be fixed by #242

**Comment by jfhbrook** [comment]
Fixed in 4.0.0
